zimbra/apache-components/zimbra-apache-components/rpm/SPECS/apache-components.spec

```diff
@@ -1,6 +1,6 @@
 Summary:            Zimbra components for apache package
 Name:               zimbra-apache-components
-Version:            2.0.5
+Version:            2.0.6
 Release:            1zimbra8.8b1ZAPPEND 
 License:            GPL-2
 Requires:           zimbra-apache-base, zimbra-httpd >= 2.4.51-1zimbra8.7b3ZAPPEND, zimbra-php >= 7.3.25-1zimbra8.7b3ZAPPEND
```

Comment:
Don't we need to change `zimbra-php >= 7.3.25-1zimbra8.7b3ZAPPEND` to 7.4.27?

Reply:
Thank you, this has been corrected